### ISSUE TITLE
An option for rendering math as image?

### ISSUE THREAD

**Issue by kindlychung** [mathjax/MathJax#1136]
A global option allowing the reader to switch the rendering between html+css and rasterized image will be very welcome.

A google search of "copy mathjax into word" shows  33,800 results, people are looking for ways to do this. Adding an option to render equations as images will make this extremely easy.

**Comment by pkra** [comment]
> option allowing the reader to switch the rendering between html+css and rasterized image 

There are no plans for this since you can already generate SVGs from MathJax. If anybody wants to build an extension, it would fit on our third party extension repository. For server-side conversion, there's https://github.com/mathjax/MathJax-node

> copy mathjax into word

An image would be the wrong way to go about -- you can use MathML instead, see https://youtu.be/VBt0Qeq5GWw for an old tutorial.

**Comment by kindlychung** [comment]
Yeah, that works but it requires you to copy and paste the MathML codes one by one.
